### PR TITLE
Jasmine 3

### PR DESCRIPTION
**Pull request by nifgraup**
- Links to documentation:https://jasmine.github.io/api/3.6/global
- Link to GitHub or NPM: https://github.com/jasmine/jasmine
- Type of contribution: addition

Other notes:

Mostly same 2.4 definitions & tests but with some improvements.

### CODE REVIEW

File: definitions/npm/jasmine_v2.4.x/flow_v0.104.x-/jasmine_v2.4.x.js [hunk 1]
```diff
@@ -1,110 +0,0 @@
-/* eslint-disable flowtype/no-weak-types */
```
Comment (pascalduez) [inline definitions/npm/jasmine_v2.4.x/flow_v0.104.x-/jasmine_v2.4.x.js:1]: Please don't delete the 2.4 definitions.

File: definitions/npm/jasmine_v3.x.x/flow_v0.125.x-/jasmine_v3.x.x.js [hunk 2]
```diff
@@ -0,0 +1,155 @@
+/* eslint-disable flowtype/no-weak-types */
```
Comment (pascalduez) [inline definitions/npm/jasmine_v3.x.x/flow_v0.125.x-/jasmine_v3.x.x.js:1]: Not 100% sure about this. It's just a warning, Well those are a lot of `any` (looks like TS) :sweat_smile:

Comment (pascalduez) [reply definitions/npm/jasmine_v3.x.x/flow_v0.125.x-/jasmine_v3.x.x.js:1]: This will ends up in installed definitions, so please remove.

Comment (nifgraup) [reply definitions/npm/jasmine_v3.x.x/flow_v0.125.x-/jasmine_v3.x.x.js:1]: Sounds like a lint rule against eslint-disable comments is needed.

Comment (nifgraup) [reply definitions/npm/jasmine_v3.x.x/flow_v0.125.x-/jasmine_v3.x.x.js:1]: I did a mass change of `Function` to `(...any) => any`.

**Review by pascalduez — APPROVED** [review]

**Pull request merged**